.github/workflows/smoke.yml

```diff
@@ -0,0 +1,112 @@
+name: Smoke Test
+on:
+  pull_request_target:
+    paths: 'task/**'
+  schedule:
+  - cron: '0 9 * * 1'
+jobs:
+  authorize:
+    environment: ${{ github.event_name == 'schedule' && 'automatic' || 'manual' }}
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo ✓
+  start:
+    needs: authorize
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    outputs:
+      kubeconfig: ${{ steps.cluster.outputs.kubeconfig }}
+    steps:
+    - id: cluster
```

Comment:
great but this probably need documenting/a tutorial somewhere in a follow-up?

Reply:
Did you mean https://www.notion.so/iterative/Setting-up-Azure-Kubernetes-Service-c20e523be5484ec29a2c2ad5fcc76be2? 😉 I wonder if we should make that document public.

Reply:
Transcribed to pure Markdown on https://github.com/iterative/terraform-provider-iterative/pull/189#issuecomment-916911104

Reply:
Moved to https://github.com/iterative/terraform-provider-iterative/issues/257